Gruntfile.js

```diff
@@ -289,7 +291,13 @@ module.exports = function(grunt) {
         base: {
           path: 'target',
           options: {
-            extensions: ['html']
+            extensions: ['html'],
+            setHeaders: res => {
+              res.setHeader(
+                'Content-Security-Policy', 
+                `script-src 'unsafe-inline' http://localhost:${DEFAULT_SERVER_PORT}`
```

Comment:
Probably OK since this is for our Widget playground, but can we handle this at a per-request bases (like we do [here](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/spec-okta-api/auth/services/devicefingerprint.js#L24)) instead of apply `unsafe-inline` globally? If we can't, mind removing the code linked to as it is no longer a required override?

Reply:
This is not for the playground, this is the server that runs the E2E test harness: `grunt connect:e2e`

Reply:
Ah - I now see that we have [different logic](https://github.com/okta/okta-signin-widget/pull/1820/files#diff-a1e1418cd078bbe08a86cd5d94caccce9c39a5db741e5fdf46311b51890dcd3bR63) for the playground. Thanks for the correction!